src/AddTodoForm.js

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+
+const AddTodoForm = () => {
+    return (
+        <form>
+            <label htmlFor="todoTitle">Title</label>
+            <input id="todoTitle"></input>
```

Comment:
🎨 In JSX, when an element doesn't have any content or children inside of it, like this input, we can skip using the closing tag and instead putting a slash at the end of the opening tag, like so:

```js
<input id="todoTitle" />
```